### PR TITLE
Add camel-pulsar wrapper

### PR DESCRIPTION
**Pull request by f2par0**
camel-pulsar wrapper
Added a restriction to the Export-Packages because it contains some shaded dependencies

### CODE REVIEW

File: features/src/main/feature/camel-features.xml [hunk 1]
```diff
@@ -1542,6 +1542,11 @@
         <bundle dependency='true'>wrap:mvn:com.pubnub/pubnub-gson/${pubnub-version}</bundle>
         <bundle>mvn:org.apache.camel.karaf/camel-pubnub/${project.version}</bundle>
     </feature>
+    <feature name='camel-pulsar' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <bundle dependency='true'>wrap:mvn:org.apache.pulsar/pulsar-client-admin/${pulsar-version}$overwrite=merge&amp;Export-Package=org.apache.pulsar*;version=${pulsar-version};resolution:=optional</bundle>
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml]: Warning, I see 2 problems with your wrap URL:

1. The jar is not an OSGI bundle so `overwrite=merge` is not needed, as it generates it for scratch
2. The resolution only makes sense for import packages

Comment (f2par0) [reply features/src/main/feature/camel-features.xml]: ok, fixed.

File: features/src/main/feature/camel-features.xml [hunk 2]
```diff
@@ -1542,6 +1542,11 @@
         <bundle dependency='true'>wrap:mvn:com.pubnub/pubnub-gson/${pubnub-version}</bundle>
         <bundle>mvn:org.apache.camel.karaf/camel-pubnub/${project.version}</bundle>
     </feature>
+    <feature name='camel-pulsar' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml]:
```suggestion
        <feature version='${camel.osgi.version.range}'>camel-core</feature>
```

**Review by essobedo — APPROVED** [review]
I added the version range

**Pull request merged**